ui/pages/home/home.component.js

```diff
@@ -285,6 +287,43 @@ export default class Home extends PureComponent {
     );
   }
 
+  getFooterContent() {
+    const { t } = this.context;
+
+    if (isBeta()) {
```

Comment:
You're probably going to hate me but in a few places in this PR we add new class methods for easily switching between two variations of content (beta vs not beta) imo this is extending a pattern that we have agreed to eliminate (class components -> functional components).

Although I'm not proposing that you refactor these components to functional ones, I think adding new methods like this increases the difficulty to refactor them later. In addition this is likely setting the stage for what is a pattern for beta build in the future. 

I'd prefer it if we either:
1) inline the content using ternaries where the content doesn't become obtuse. 
2) create new components where content is incredibly different.

home-footer-beta.js and home-footer.js. I don't know if we support dynamic imports and code splitting yet (@kumavis ?) but we could utilize this pattern in the future to completely extract beta content from production builds and vice versa.  For now, I think its fine to import both normally and render one or the other.